float32 UCDieTemperature
float32 oilTemperature
